src/editor/reducers/notebook-reducer.js

```diff
@@ -103,6 +103,15 @@ const notebookReducer = (state = newNotebook(), action) => {
       });
     }
 
+    case "REVERT_TO_NOTEBOOK_HISTORY_BROWSER_SELECTED_REVISION_ID": {
```

Comment:
There should be no need to define a specific action for this. You should be able to use the initial `UPDATE_IOMD_CONTENT` action (exposed via the `updateIomdContent` function)

Reply:
Done